### PR TITLE
fix handle cleanlogs

### LINKED ISSUE #51: execute a command like GetCleanLogs() and get the output
while working on the CLI version I tried to listen on an event with `bot.cleanLogsEvents.subscribe(on_clean_event)` and then executed `GetCleanLogs()` ... I was able to see all the clean info in the debug info but on_clean_event is never executed because:
```
deebotozmo.vacuum_bot WARNING  Invalid Event cleanlogs: {}
```

edit: same user as @c79

### PR DESCRIPTION
**Pull request by edenhaus**
Fixes #51

### CODE REVIEW

File: deebotozmo/vacuum_bot.py [hunk 1]
```diff
@@ -179,8 +183,6 @@ async def handle(self, event_name: str, event: dict, requested: bool = True) ->
                 await self._handle_charge_state(event_data)
         elif event_name == "lifespan":
             await self._handle_life_span(event_data)
-        elif event_name == "cleanlogs":
```
Comment (rany2) [inline deebotozmo/vacuum_bot.py:182]: Isn't there a chance that we get a cleanlogs event via mqtt? I don't think this should be removed. I already added a `return` statement to end execution of the rest of the function in fact it was `requested == True`

Comment (edenhaus) [reply deebotozmo/vacuum_bot.py:182]: Clean events can only be requested as they are not stored directly on the robot but an the ecovacs servers. Therefore it's fine to move it

**Comment by edenhaus** [comment]
@rany2 Have you tested this change?

**Comment by rany2** [comment]
@edenhaus yes

**Pull request merged**